dev/docker-compose-sdp-anchor.yml

```diff
@@ -133,6 +133,22 @@ services:
                 "max_amount": 10000
               },
               "withdraw": {"enabled": false}
+            },
```

Comment:
Do we want to add XLM to `values.yaml` and `docker-compose-e2e-tests.yml` as well ?

Reply:
We do want to add it to the `values.yaml`. But for the `docker-compose-e2e-tests.yml`, I don't think so. Thank you for pointing that out.

Reply:
changed: [afa7ef0](https://github.com/stellar/stellar-disbursement-platform-backend/pull/1/commits/afa7ef088e18ea3d58a677a2a3ff2bf06b2720d8)